### PR TITLE
added-masonry-layout-file

### PR DESCRIPTION
**Pull request by murtazshaikh**
review pr...

### CODE REVIEW

**Review by OSAMA478 — APPROVED** [review]
Nice...

**Review by anask601 — APPROVED** [review]

**Review by syedabdullahali — APPROVED** [review]

File: style.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+}
+.container{
+    position: relative;
+
+    max-width: 95%;
+
+    margin: 40px;
+
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
```
Comment (Abbas4tech) [inline style.css:28]: What does this `auto-fill` affect ??? @murtazshaikh

File: style.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    grid-template-rows: minmax(100px, auto );
+    grid-auto-flow: dense;
```
Comment (Abbas4tech) [inline style.css:30]: Can you explain what will it affect @murtazshaikh

File: style.css [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+.container .box img{
+    position: relative;
+
+    max-width: 100px;
+    
+    margin-bottom: 10px;
+}
+.container .box:nth-child(1){
+    grid-column: span 2;
+    grid-row: span 1;
```
Comment (Abbas4tech) [inline style.css:59]: What is the meaning of writing `span` here @murtazshaikh

**Review by Abbas4tech — CHANGES_REQUESTED** [review]

**Comment by murtazshaikh** [comment]
> Nice...

Thankyou @OSAMA478

**Pull request merged**